dicodile/utils/segmentation.py

```diff
@@ -85,7 +85,7 @@ def compute_n_seg(self):
         self.n_seg_per_axis = []
         for size_ax, size_seg_ax in zip(self.signal_support, self.seg_support):
             # Make sure that n_seg_ax is of type int (and not np.int*)
-            n_seg_ax = max(1, int(size_ax // size_seg_ax))
+            n_seg_ax = max(1, int(size_ax // size_seg_ax - 1))
```

Comment:
Actually, are you sure this covers for all the coordinates?

Reply:
I've updated segmentation in accordance with alphacsc. Could you review the recent changes?

Reply:
This does not fix the issue, I need to work further on that. Please ignore my previous comment.